logpolicy/logpolicy.go

```diff
@@ -40,54 +48,66 @@ func (c *Config) ToBytes() []byte {
 	return data
 }
 
-func (c *Config) Save(statefile string) {
+// Save writes the JSON representation of c to stateFile.
+func (c *Config) Save(stateFile string) error {
```

Comment:
Do you have a separate PR to use this error so it isn't lost?

Reply:
The only caller I see is also updated in this PR already.